javaharness/java/arcs/android/BUILD

```diff
@@ -15,11 +15,12 @@ android_library(
     ],
     assets_dir = "",
     exports_manifest = 1,
-    idl_srcs = glob(["*.aidl"]),
+    # idl_srcs = glob(["*.aidl"]),
```

Comment:
Rather than keeping this around, can it just be deleted?

Reply:
removed